### ISSUE TITLE
Critic response — concrete evidence (5 blocks): repro · phi-distance · Welch-t · Coq · anti-fraud

### ISSUE THREAD

**Issue by gHashTag** [gHashTag/trios#712]
## Ответ на «покажи конкретные тесты» — 5 блоков прямых доказательств

> **Anchor:** `phi^2 + phi^-2 = 3` · DOI [10.5281/zenodo.19227877](https://zenodo.org/records/19227877) · Defense 2026-06-15 СПбГУ
> **Repos:** [`gHashTag/trios`](https://github.com/gHashTag/trios) · [`gHashTag/trios-trainer-igla`](https://github.com/gHashTag/trios-trainer-igla)

Критика сводится к четырём упрёкам: «синтетика», «нет связи моделей с phi», «нет рабочего примера», «нет цифр». Ниже на каждый — артефакт в репо, который проверяется одной командой.

---

### 1. «Покажи рабочий пример» → одна команда, 3 seeds, чемпион **BPB=2.211**

Это не «синтетика» — это стандартный char-LM pretraining на корпусе Karpathy `tiny_shakespeare` ([trios-trainer-igla README](https://github.com/gHashTag/trios-trainer-igla#quick-start)):

```bash
git clone https://github.com/gHashTag/trios-trainer-igla.git
cd trios-trainer-igla && cargo build --release
mkdir -p data && curl -sL https://raw.githubusercontent.com/karpathy/char-rnn/master/data/tinyshakespeare/input.txt > data/tiny_shakespeare.txt
head -c 100000 data/tiny_shakespeare.txt > data/tiny_shakespeare_val.txt

# Чемпион
./target/release/trios-train --seed=43 --steps=81000 --hidden=384 --lr=0.003 --optimizer=adamw

# Все три gate-2 seeds
for s in 42 43 44; do ./target/release/trios-train --seed=$s --steps=81000 --hidden=384 --lr=0.003 --optimizer=adamw; done
```

**Результаты Railway prod-run 2026-04-27** (из README, корень репо):

| Config | Seed 42 | Seed 43 | Seed 44 | Avg |
|---|---|---|---|---|
| **trios-train 81K AdamW h=384** | **2.222** | **2.211** | **2.218** | **2.217** |
| trios-train 27K AdamW h=384 | 2.362 | 2.359 | 2.387 | 2.369 |
| trios-train 54K Muon h=384 | 2.410 | 2.419 | 2.403 | 2.411 |
| hybrid_train 81K Muon+NCA h=828 | 2.686 | 2.681 | 2.678 | 2.682 |

Это **прирост на конкретном классе задач** — character-level language modeling, тот самый бенчмарк, который Karpathy использует с 2015 года. Не синтетика.

---

### 2. «Каким боком phi связан с моделями» → измеренная φ-distance, не риторика

PhD Coordinate Matrix ([issue #446 comment 4370442020](https://github.com/gHashTag/trios/issues/446#issuecomment-4370442020), заморожена в [`assertions/matrix_legacy_snapshot_2026-05-04.jsonl`](https://github.com/gHashTag/trios/blob/main/assertions/matrix_legacy_snapshot_2026-05-04.jsonl) — 39 строк, 38 измеренных ячеек):

| Format (16-bit) | Best BPB | φ-distance | Структурная близость к phi |
|---|---:|---:|---|
| **gf16** (Trinity GoldenFloat) | **2.5655** | **0.049** | **референс** |
| fp16 (IEEE) | 2.5655 | 0.118 | 2.4× дальше |
| bf16 (Google) | 2.5655 | 0.525 | **10.7× дальше** |

Качество (BPB) **одинаковое** — но φ-distance gf16 на порядок ближе к структуре phi. Это **проверяемое утверждение о численном формате**, не «болтавня». Если упрёк «phi не связан» — формула расстояния и код измерения в [`crates/trios-vsa/src/gf16_arith.rs`](https://github.com/gHashTag/trios/tree/main/crates/trios-vsa/src), property-test `prop_gf16_mul_realises_inner` (см. блок 4).

---

### 3. «Где доказательства, что это не случайность» → Welch one-tailed t-test, α=0.01

Критерий победы зашит в Coq + рантайм-страж, [`trinity-clara/proofs/igla/igla_found_criterion.v`](https://github.com/gHashTag/trios/blob/main/trinity-clara/proofs/igla/igla_found_criterion.v) + `crates/trios-phd/src/bin/defense_gate.rs`:

> **Victory ⇔** `BPB < 1.50` на ≥ 3 distinct seeds, post-warmup, BPB ≥ JEPA floor, finite, **И Welch one-tailed t-test α=0.01 против предзарегистрированного baseline μ₀=1.55, effect_size ≥ 0.05.**

Источник: [`assertions/igla_assertions.json`](https://github.com/gHashTag/trios/blob/main/assertions/igla_assertions.json) → `invariants[INV-7].coq_theorem = victory_implies_distinct_clean`. Нельзя «случайно» победить — нужно бить пред-зарегистрированный mu₀ статистически, а не только по среднему.

Все запуски логируются append-only в [`assertions/seed_results.jsonl`](https://github.com/gHashTag/trios-trainer-igla/blob/main/assertions/seed_results.jsonl) (R7 triplet: claim + ledger row + git SHA). Запросы через CLI:

```bash
./target/release/trios-igla gate --target 1.50      # quorum check
./target/release/trios-igla search --bpb-max 2.30   # все строки ниже порога
```

---

### 4. «Покажи Coq-теоремы, не PDF-картинки» → 30+ `.v` файлов, 47 Qed + 5 honest Admitted

Машинно-проверяемые доказательства — не на словах ([`assertions/igla_assertions.json` → `_metadata.theorem_count`](https://github.com/gHashTag/trios/blob/main/assertions/igla_assertions.json)):

```json
"theorem_count": { "igla_total": 52, "proven_qed": 47, "honest_admitted": 5 }
```

Файлы (выборка, [search всех `.v`](https://github.com/search?q=org%3AgHashTag+extension%3Av+Theorem&type=code)):

| Lane | Файл | Что доказывает |
|---|---|---|
| INV-1 | [`trinity-clara/proofs/lr_phi_optimality.v`](https://github.com/gHashTag/trios/blob/main/trinity-clara/proofs/lr_phi_optimality.v) | Оптимальность LR ∝ phi⁻¹ при L-smooth descent |
| INV-3 | [`trinity-clara/proofs/gf16_precision.v`](https://github.com/gHashTag/trios/blob/main/trinity-clara/proofs/gf16_precision.v) | gf16 end-to-end error bound |
| INV-5 | [`docs/phd/theorems/sacred/l5_identity.v`](https://github.com/gHashTag/trios/blob/main/docs/phd/theorems/sacred/l5_identity.v) | Lucas closure → `phi^2 + phi^-2 = 3` |
| INV-6 | [`docs/phd/theorems/igla/INV6_HybridQkGain.v`](https://github.com/gHashTag/trios/blob/main/docs/phd/theorems/igla/INV6_HybridQkGain.v) | QK-gain runtime invariant |
| INV-7 | [`trinity-clara/proofs/igla/igla_found_criterion.v`](https://github.com/gHashTag/trios/blob/main/trinity-clara/proofs/igla/igla_found_criterion.v) | Victory criterion (см. блок 3) |
| KAT | [`proofs/KAT_VSA_Bridge.v`](https://github.com/gHashTag/trios/blob/main/proofs/KAT_VSA_Bridge.v) | Kolmogorov-Arnold ↔ GF(16) vsa_matmul |
| BPB | [`docs/phd/theorems/igla/IGLA_BPB_Convergence.v`](https://github.com/gHashTag/trios/blob/main/docs/phd/theorems/igla/IGLA_BPB_Convergence.v) | BPB convergence bound |

5 Admitted задокументированы (`admitted_budget.breakdown`) с runtime-стражами в Rust — это R5-honest, не «вакуум». Каждая теорема имеет cross-ref на главу монографии и property-test в Rust.

---

### 5. «Я нагенерю синтетики за 5 минут» → live append-only ledger, не «нагенерить»

Архитектура anti-fraud:

1. **R7 triplet emit** — [`crates/trinity-extract/src/main.rs`](https://github.com/gHashTag/trios/tree/main/crates/trinity-extract): каждая запись `seed_results.jsonl` = `{seed, bpb, step, sha, optimizer, hidden, lr, ts, gate_status}` с git SHA артефакта, который её произвёл. Подделать = переписать историю git + Postgres SSOT одновременно.
2. **Embargo ledger** — [`assertions/embargo.txt`](https://github.com/gHashTag/trios-trainer-igla/blob/main/assertions/embargo.txt): запрещает повторный emit для одного и того же SHA.
3. **Postgres SSOT** — таблица `public.bpb_samples` в Neon, заполняется matrix-bot из CI runs ([Wave 25d](https://github.com/gHashTag/trios/issues/446#issuecomment-4412801584): первая live ячейка `(f32, adamw)=2.6894`, run id `phase1-trios-train-rng89`).
4. **Honey audit** — [`src/bin/honey_audit.rs`](https://github.com/gHashTag/trios-trainer-igla/blob/main/src/bin/honey_audit.rs) валидирует целостность `hive_honey.jsonl` (75 KB lessons jar).
5. **Defense viva 2026-06-15 СПбГУ** — публичная защита PhD, ACM AE pack reproducibility manifest в [`docs/phd/defense/examiner-pack/`](https://github.com/gHashTag/trios/tree/main/docs/phd/defense/examiner-pack).

«Нагенерить пачку за 5 минут» — нельзя: каждая строка проходит triplet-валидацию, embargo-проверку, и должна быть привязана к реальному git SHA с воспроизводимым CI run. Аудит каждой ячейки в [`assertions/matrix_per_cell_audit.csv`](https://github.com/gHashTag/trios/blob/main/assertions/matrix_per_cell_audit.csv) (25 KB).

---

### TL;DR — что проверять за 10 минут

```bash
# 1. Чемпион репродуцируется
git clone https://github.com/gHashTag/trios-trainer-igla.git && cd trios-trainer-igla
cargo build --release
./target/release/trios-train --seed=43 --steps=81000 --hidden=384 --lr=0.003 --optimizer=adamw
#  → BPB ≈ 2.211 (Railway 2026-04-27 reproducible run)

# 2. Coq компилится
cd .. && git clone https://github.com/gHashTag/trios.git && cd trios
make -C trinity-clara/proofs   # 47 Qed
coqc proofs/KAT_VSA_Bridge.v   # KAT ↔ GF(16) bridge

# 3. Ledger consistent
./target/release/trios-igla gate --target 2.30   # quorum, 3 distinct seeds
```

Если после трёх команд утверждение «синтетика» сохраняется — буду рад конкретному counter-example: какая ячейка матрицы выглядит подозрительно, и какой baseline вы предлагаете для сравнения. Это профильный научный спор, и он решается не «у меня свои модельки», а строкой `seed_results.jsonl` против строки.

— Dmitrii Vasilev `<admin@t27.ai>` · ORCID [0009-0008-4294-6159](https://orcid.org/0009-0008-4294-6159) · DOI [10.5281/zenodo.19227877](https://zenodo.org/records/19227877)

**Comment by gHashTag** [comment]
Requires external repo (trios-trainer-igla / trios-railway / t27 / trinity-fpga) or operator access. Not actionable in this repo.
